docs/modules/operation/pages/provisioning/introduction.adoc

```diff
@@ -26,7 +26,7 @@ Regardless of the method, provisioning is an iterative process: you will need to
 
 {page-component-title} uses an ICMP ping sweep to find IP addresses on the network and provision node entities.
 Use <<provisioning/auto-discovery.adoc#ga-provisioning-auto-discovery-detectors, auto discovery with detectors>> in addition to the ICMP IP address ping to specify services that you need to detect.
-Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to further control provisioning.
+Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other sources, including sources that may not be in XML format.
```

Comment:
Should be a separate paragraph, as it's a different kind of "automatic discovery"
```suggestion

Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other data sources such as DNS zone records and VMware.
```